README.md

```diff
@@ -1,36 +1,36 @@
 <h1 align="center">
-<img src="/assets/icon.png" alt="Icon" width="100" height="100" </img>
+<img src="/assets/icon.png" alt="Icon" style="width:100px;height:100px;"/>
 <br>
 YouTube Spammer Purge
 <br>
 </h1>
 
 <div align="center">
   <a href="https://github.com/ThioJoe/YT-Spammer-Purge/issues">
-    <img src="https://img.shields.io/github/issues/ThioJoe/YT-Spammer-Purge" />
+    <img alt="Issue count" src="https://img.shields.io/github/issues/ThioJoe/YT-Spammer-Purge" />
   </a>
   <a href="https://github.com/ThioJoe/YT-Spammer-Purge/pulls">
-    <img src="https://img.shields.io/github/issues-pr/ThioJoe/YT-Spammer-Purge" />
+    <img alt="Pull Request count" src="https://img.shields.io/github/issues-pr/ThioJoe/YT-Spammer-Purge" />
   </a>
   <a href="https://github.com/ThioJoe/YT-Spammer-Purge/releases">
-    <img src="https://img.shields.io/github/v/release/ThioJoe/YT-Spammer-Purge?include_prereleases&label=Latest%20Release" />
+    <img alt="Current release version" src="https://img.shields.io/github/v/release/ThioJoe/YT-Spammer-Purge?include_prereleases&label=Latest%20Release" />
   </a>
   <a href="https://github.com/ThioJoe/YT-Spammer-Purge/actions/workflows/codeql-analysis.yml">
-    <img src="https://github.com/ThioJoe/YT-Spammer-Purge/actions/workflows/codeql-analysis.yml/badge.svg" />
+    <img alt="Code Quality status" src="https://github.com/ThioJoe/YT-Spammer-Purge/actions/workflows/codeql-analysis.yml/badge.svg" />
   </a>
   <a href="https://github.com/ThioJoe/YT-Spammer-Purge/actions/workflows/docker-build.yml">
-    <img src="https://github.com/ThioJoe/YT-Spammer-Purge/actions/workflows/docker-build.yml/badge.svg" />
+    <img alt="Build status" src="https://github.com/ThioJoe/YT-Spammer-Purge/actions/workflows/docker-build.yml/badge.svg" />
   </a>
 </div>
 
-**What Is This?** - Allows you to filter and search for spammer comments on your channel and other's channel(s) in many different ways AND delete/report them all at once (see features below).
+**What Is This?** - Allows you to filter and search for spammer comments on your channel and other's channel(s) in many ways AND delete/report them all at once (see features below).
```

Comment:
I mean, different kinda adds some effect, I would keep it as is, but there's no harm if you remove the word too, but just my opinion.